HeliPort/Appearance/StatusMenu.swift

```diff
@@ -491,6 +495,16 @@ final class StatusMenu: NSMenu, NSMenuDelegate {
                 self.phyModeItem.title = NSLocalizedString("    PHY Mode: ", comment: "") + phyMode
                 self.mcsIndexItem.title = NSLocalizedString("    MCS Index: ", comment: "") + mcsIndex
                 self.nssItem.title = NSLocalizedString("    NSS: ", comment: "") + nss
+                if let wifiItemView = self.networkItemList[0].view as? WifiMenuItemView {
```

Comment:
I think adding a guard here and checking if isNetworkEnabled is still true since it still shows the Wi-Fi item even after it's set as "off".